app/ide-desktop/lib/client/node_modules/README.md

```diff
@@ -1,10 +0,0 @@
-Placeholder to trick `electron-builder` into thinking that its dependencies are
-installed.
-
-`electron-builder` checks for presence of `node_modules` directory. If it is not
-present, it will install dependencies with `--production` flag (erasing all 
-dev-only dependencies). This does not work sensibly with NPM workspaces. We have
-our `node_modules` in the root directory, not here.
-
-Without this workaround, `electron-builder` will end up erasing its own
-dependencies and failing because of that.
```

Comment:
This file should be not removed from the repo. It is deleted sometimes erroneously by build scripts (@mwu-tow is aware of this).

Reply:
Actually I already removed this workaround, AFAIK the file is not present on `develop`.